src/include/rados/librados.hpp

```diff
@@ -14,6 +14,18 @@
 #include "librados_fwd.hpp"
 #include "rados_types.hpp"
 
+#define WITH_JAGER //will be removed after testing
+
+#ifdef WITH_JAGER
+  #include "../../common/tracer.h"
+  #define __FILENAME__ (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__)
+#else
+  typedef char Span;
```

Comment:
I feel this is a workaround, as we discussed yesterday adding a parent_span to some common structure instead of changing librados API could be done here, which shall eliminate this

Reply:
@ideepika if make Span a datamember in librados call then, I will make a new constructor to initialize that Span, a getter method to get that span, and also the objects creation of librados needs to be changed in some places, and even after doing the above imports an typedef will still be there, should I go for it?